multiplayer/src/util/index.ts

```diff
@@ -80,7 +80,7 @@ export function cleanupJoinCode(
     joinCode: string | undefined
 ): string | undefined {
     if (!joinCode) return undefined;
-    joinCode = joinCode.trim();
+    joinCode = joinCode.replaceAll(" ", "");
```

Comment:
Oh I see, this will remove the space. Maybe it's fine.

Reply:
Should we maybe just move line 85 up and delete this / line 84? That should handle this just fine as it's already clearing out all non alphanumeric (which would fix if there's a weird nbsp or something)